SConstruct

```diff
@@ -1,49 +1,59 @@
-import errno, os, shutil
-
-user_libs = ['libboost_date_time.so.1.54.0',
-             'libboost_filesystem.so.1.54.0',
-             'libboost_iostreams.so.1.54.0',
-             'libboost_regex.so.1.54.0',
-             'libboost_system.so.1.54.0',
-             'libxerces-c-3.1.so']
+import os
+import shutil
+import subprocess
+import sys
+
+linux_boost_version = '1.54.0'
+
+libs = ['libboost_date_time',
+             'libboost_filesystem',
+             'libboost_iostreams',
+             'libboost_regex',
+             'libboost_system']
```

Comment:
`libs` is overwritten in the `darwin` case so shouldn't this just be inside the `linux2` branch?

Reply:
In the darwin case, libs uses these names to build absolute paths to the static libraries to be linked.